app/components/contact_details_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render(SummaryCardComponent, rows: @contact_details_form_rows) %>
```

Comment:
I believe you can simply call:

```ruby
<%= render(SummaryCardComponent, rows: convert_to_rows) %>
```

If so, `convert_to_rows` can be called `contact_details_form_rows` so it makes more sense in this use.